Description: Use only stuff included in linux-kbuild
 This patch removes all calls of files that are only included
 in linux-headers. Since there is no single linux-headers
 package that can be used as Build-Depends, the unnecessary
 calls of files from this packages are removed.
Author: Jan Luca Naumann <j.naumann@fu-berlin.de>
Forwarded: not-needed
Last-Update: 2016-08-31
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/Makefile
+++ b/Makefile
@@ -23,7 +23,7 @@
 all: aufs.ko usr/include/linux/aufs_type.h
 
 clean:
-	${MAKE} ${MakeMod} $@
+#	${MAKE} ${MakeMod} $@
 	find . -type f -name '*~' | xargs -r ${RM}
 	${RM} -r aufs.ko usr
 
@@ -45,6 +45,5 @@
 usr/include/linux/aufs_type.h:
 	${MAKE} -rR -C ${KDIR} \
 		-f scripts/Makefile.headersinst \
-		-f Makefile \
 		obj=${d}/include/uapi/linux dst=${d}/usr/include/linux
 	test -s $@
